cmd/spire-agent/cli/run/run_test.go

```diff
@@ -432,6 +434,16 @@ func TestMergeInput(t *testing.T) {
 				require.Equal(t, "foo", c.Agent.TrustBundlePath)
 			},
 		},
+		{
```

Comment:
can you add a test to verify it is not possible to have it configured by url and path?